extensions/wikia/Search/classes/MediaWikiService.php

```diff
@@ -909,7 +907,7 @@ protected function getPageFromPageId( $pageId ) {
 		$page = \Article::newFromID( $pageId );
 
 		if ( $page === null ) {
-			return null;
+			throw new StaleResultException( $pageId );
```

Comment:
Let's cast `$pageId` to string or we may loose this message in ELK, due to elasticsearch type checks.